vividus-soft-assert/build.gradle

```diff
@@ -4,6 +4,7 @@ dependencies {
     api(group: 'org.hamcrest', name: 'hamcrest', version: '2.2')
 
     implementation project(':vividus-test-context')
+    implementation(group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0')
```

Comment:
it's better to create own simple interface instead of adding extra dependencies to low-level modules